GSA-TTS/FAC
# Bump PDF Character Minimum to 6000 Characters

# Bump PDF Character Minimum to 6000 Characters

Issue: https://github.com/GSA-TTS/FAC/issues/3237

Testing against production, I downloaded 500 semi-random PDFs. Three fail on this new character count check, because they have only a little text and a lot of images. That's something we want to prevent, so this change is good.

```
2023-06-GSAFAC-0000002047
2022-12-GSAFAC-0000001186
2023-03-GSAFAC-0000000021
```

## Changes:
Instead of looking for a non-zero character count on PDF upload, check for more than 6000 characters. See issue for more info on why this value was selected. 

Also, update the `basic.pdf` used in tests so they don't start failing.

## How to test:
1. Switch to this branch and run normally.
2. Start a submission, or navigate to an ongoing one.
3. Try to upload a too-small PDF, and verify that it fails.
4. Try to upload an appropriately-sized PDF, and verify it passes. 

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
